util/env_posix.cc

```diff
@@ -874,7 +874,7 @@ class SingletonEnv {
 #endif  // !defined(NDEBUG)
     static_assert(sizeof(env_storage_) >= sizeof(EnvType),
                   "env_storage_ will not fit the Env");
-    static_assert(std::is_standard_layout_v<SingletonEnv<EnvType>>);
+    static_assert(std::is_standard_layout<SingletonEnv<EnvType>>::value);
```

Comment:
You likely need the message as well to be C++11 compatible
```suggestion
    static_assert(std::is_standard_layout<SingletonEnv<EnvType>>::value,
                    "SingletonEnv<EnvType> is not standard layout");
```

Reply:
See: https://github.com/google/leveldb/pull/1251/files

Reply:
ok, haven't seen it before, close this now.